sphinx/templates/latex/longtable.tex_t

```diff
@@ -0,0 +1,21 @@
+\begin{longtable}<%= table.get_colspec() %>
```

Comment:
there is possibility in future to add `[c|l|r]` option here for horizontal alignment. (new ``:align:`` option of Docutils 0.13)